tests/test_FLEbasis2D.py

```diff
@@ -70,8 +71,10 @@ def relerr(base, approx):
 
 @pytest.mark.parametrize("basis", test_bases, ids=show_fle_params)
 class TestFLEBasis2D(UniversalBasisMixin):
-    # Loosen the tolerance for `cufinufft` to be within 15%
-    test_eps = 1.15 if backend_available("cufinufft") else 1.0
+    # Loosen the tolerance for `cufinufft` and `osx_arm` to be within 15%
+    test_eps = 1.0
+    if backend_available("cufinufft") or platform.system() == "Darwin":
+        test_eps = 1.16
```

Comment:
Was this intentionally changed to 1.16 from 1.15? Still reads 15% above so wasn't sure if this was a typo.

Reply:
Hmm, looks like this test failed CI.